.github/sync-files.yaml

```diff
@@ -0,0 +1,24 @@
+- repository: autowarefoundation/autoware
+  files:
+    - source: .github/dependabot.yaml
+    - source: .github/workflows/pre-commit-optional.yaml
+    - source: .github/workflows/semantic-pull-request.yaml
+    - source: .github/workflows/spell-check.yaml
+    - source: .clang-format
+    - source: .markdown-link-check.json
+    - source: .markdownlint.yaml
+    - source: .pre-commit-config-optional.yaml
+    - source: .prettierignore
+    - source: .prettierrc
+    - source: .yamllint.yaml
+    - source: CPPLINT.cfg
+    - source: setup.cfg
+
+- repository: autowarefoundation/autoware_common
+  files:
+    - source: .github/workflows/automatic-rebase.yaml
```

Comment:
I've added this to autowarefoundation/autoware, so let's copy from here.
https://github.com/autowarefoundation/autoware/blob/tier4/proposal/.github/workflows/automatic-rebase.yaml

Reply:
https://github.com/tier4/ros2-project-template/pull/2/files#r799262074
https://github.com/tier4/ros2-project-template/pull/2/files#r799262145